src/client/router.js

```diff
@@ -45,6 +45,18 @@ module.exports = () => {
           });
           return h(Features.Edit, editProps);
         }
+      },
+      {
+        path: '/gq',
+        render: props => h(Features.GeneQuery, props)
+      },
+      {
+        path: '/enrich',
```

Comment:
Its probably better to have one route defined.  e.g path: '/enrichment-services'.  You can use react tabs to let the testers switch between genequery/enrichment/whatever, which is easier than typing in different urls to do different things.

Reply:
https://github.com/reactjs/react-tabs for usage.  You can also look at some other files in the code base that use react tabs

Reply:
two services were merged in /gqenrich